doc/api/http.md

```diff
@@ -102,8 +102,8 @@ added: v0.3.4
   Can have the following fields:
   * `keepAlive` {Boolean} Keep sockets around in a pool to be used by
     other requests in the future. Default = `false`
-  * `keepAliveMsecs` {Integer} When using HTTP KeepAlive, how often
-    to send TCP KeepAlive packets over sockets being kept alive.
+  * `keepAliveMsecs` {Integer} When using HTTP KeepAlive, the initial delay 
```

Comment:
Agreed, tried to make it more similar to `socket.setKeepAlive`

> Set initialDelay (in milliseconds) to set the delay between the last data packet received and the first keepalive probe. Setting 0 for initialDelay will leave the value unchanged from the default (or previous) setting. Defaults to 0. 

The main thing that should be changed is that its not related to _how often_ it sends, but how long the idle is, as we see [here](https://github.com/libuv/libuv/blob/1a96fe33343f82721ba8bc93adb5a67ddcf70ec4/src/unix/tcp.c#L327) it refers to `TCP_KEEPIDLE` in unix, not `TCP_KEEPINTVL`